packages/editor-ui/src/components/WorkflowSelectorParameterInput/WorkflowSelectorParameterInput.vue

```diff
@@ -176,8 +204,36 @@ watch(
 onClickOutside(dropdown, () => {
 	isDropdownVisible.value = false;
 });
-</script>
 
+const onAddResourceClicked = () => {
+	const subWorkflowNameRegex = /My\s+Sub-Workflow\s+\d+/;
+
+	const urlSearchParams = new URLSearchParams();
+
+	if (projectStore.currentProjectId) {
+		urlSearchParams.set('projectId', projectStore.currentProjectId);
+	}
+
+	const sampleSubWorkflows = workflowsStore.allWorkflows.filter(
```

Comment:
I am slightly worried by the performance of this if there are a lot of workflows + flakiness of using regex. But ok to keep if if we really want to have unique names.

Reply:
I do not think this will turn into a issue.